templates/Package_summary.ss

```diff
@@ -1,4 +1,13 @@
 <a href="https://addons.silverstripe.org/add-ons/$Name.ATT" title="<%t SiteSummary.AddonsLinkTitle "View {package} on addons.silverstripe.org" package=$Title.ATT %>" class="package-summary__anchor" target="_blank" rel="noopener">
     <strong class="package-summary__title">$Title.XML</strong>
-    <% if $Description %><span class="package-summary__description">$Description.XML</span><% end_if %>
+    
+    <% if $Badges %>
+    <% loop $Badges %>
+        <span class="package-summary__badge<% if $Type %> package-summary__badge--$Type.ATT<% end_if %>">$Title</span>
```

Comment:
May as well add `$Title.XML`, unless it's already escaped

Reply:
Good call.